timeseries/src/autogluon/timeseries/evaluator.py

```diff
@@ -174,6 +191,10 @@ def _wape(self, y_true: pd.Series, predictions: TimeSeriesDataFrame) -> float:
         abs_target_sum = y_true.abs().sum()
         return abs_error_sum / abs_target_sum
 
+    def _rmsse(self, y_true: pd.Series, predictions: TimeSeriesDataFrame) -> float:
+        y_pred = self._get_median_forecast(predictions)
```

Comment:
I think we should use the "mean" forecast here since RMSSE is minimized by the conditional mean, not median (similar to RMSE, as described [here](https://otexts.com/fpp3/accuracy.html#scale-dependent-errors))

Reply:
Agreed and updated!